### ISSUE TITLE
updateAuthUser for MFA not working

### ISSUE THREAD

**Issue by zabakala** [firebase/firebase-admin-node#2285]
Hello. Discovered the following issue using firebase admin v10. I can successfully enroll a user for the MFA process by calling admin.auth().updateUser(...) with multiFactor/enrolledFactors provided. But removing enrolled factors is not successful (https://firebase.google.com/docs/auth/admin/manage-mfa-users):

```
admin.auth().updateUser({
  uid: '123456789',
  multiFactor: {
    enrolledFactors: null,
  },
})
.then((userRecord) => {
  console.log(userRecord.multiFactor);
})
.catch((error) => {
  console.log(error);
});
```
The factors remain in DB. I can see the same discussed for Golang as well here https://github.com/firebase/firebase-admin-go/pull/530/files.

**Comment by pragatimodi** [comment]
Hi, thanks for reporting this issue. Please use an empty array to update the MFA config for now while we fix this. 
```
admin.auth().updateUser(uid: '123456789', {
  multiFactor: {
    enrolledFactors: [],
  },
})
.then((userRecord) => {
  console.log(userRecord.multiFactor);
})
.catch((error) => {
  console.log(error);
});
```

**Comment by zabakala** [comment]
I tried with an empty array, but the without success as well.

**Comment by pragatimodi** [comment]
Is UID being entered as a method param in your code? I think the documentation needs to be amended.

**Comment by zabakala** [comment]
Yes, it is shaped like this https://github.com/firebase/firebase-admin-node/issues/2285#issue-1856072088 , but be it _null_ or an empty array _[]_ for the _enrolledFactors_ body part, it keeps having the previously assigned payload, i.e it does not get reset.

**Comment by pragatimodi** [comment]
Is `uid` entered as a method param rather than the UpdateRequest body?

**Comment by zabakala** [comment]
yes , there is also an attempt to enter it as a param.

**Comment by pragatimodi** [comment]
@zabakala I am unable to reproduce this issue, please feel free to provide more details you think may be useful to repro this. 
Also consider filing a bug report through https://firebase.google.com/support so we can track this issue internally.
